src/service/access.js

```diff
@@ -21,6 +21,7 @@ function updateIp (user, ip) {
   }
 }
 
+//DEPRECATED. Please use checkAccessFast
 function checkAccess (req, res, next) {
   var ip = req.headers["x-forwarded-for"] || req.connection.remoteAddress;
```

Comment:
Please add "DEPRECATED" logged, so we can clean up when deprecated calls disappear from logs